draft-irtf-cfrg-opaque.md

```diff
@@ -1272,23 +1230,29 @@ server functions, respectively.
 
 ~~~
 struct {
+  CredentialRequest credential_request;
   uint8 client_nonce[Nn];
   uint8 client_keyshare[Npk];
-} AuthInit;
+} KE1;
 ~~~
 
+credential_request: a CredentialRequest structure.
+
 client_nonce : A fresh randomly generated nonce of length Nn.
 
 client_keyshare : Client ephemeral key share of fixed size Npk.
 
 ~~~
 struct {
+  CredentialResponse credential_response;
   uint8 server_nonce[Nn];
   uint8 server_keyshare[Npk];
   uint8 server_mac[Nm];
-} AuthResponse;
+} KE2;
 ~~~
 
+credential_response: a CredentialResponse structure.
```

Comment:
evaluated_message: A serialized OPRF evaluated element of fixed size Noe.

masking_nonce: A nonce used for the confidentiality of the masked_response field of fixed size Nn.

masked_response: An encrypted form of the server's public key and the client's `Envelope` structure of fixed size Npk + Ne.